requirements/production.txt

```diff
@@ -1,3 +1,4 @@
 requests
+# requires: krb5-devel kobo-rpmlib
```

Comment:
Kobo should be installable directly from PyPI. Maybe just list it as a requirement?

Reply:
Test it again, no kobo-rpmlib dependency here, remove this part.